src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Products/Blocks/Tab/basic_settings_tab.html.twig

```diff
@@ -0,0 +1,136 @@
+              </span>
+            </h2>
+            {{ form_widget(productForm.basic_settings.with_combinations) }}
+            <div id="product_type_combinations_shortcut">
+                <span class="small font-secondary">
+                  {{ "Advanced settings in [1][2]Combinations[/1]"|trans({}, 'Admin.Catalog.Help')|replace({'[1]': '<a href="#tab-step3" onclick="$(\'a[href=\\\'#step3\\\']\').tab(\'show\');" class="btn sensitive px-0">', '[/1]': '</a>', '[2]': '<i class="material-icons">open_in_new</i>'})|raw }}
+                </span>
+            </div>
+          </div>
+        {% endif %}
+
+        <div class="form-group mb-4">
+          <h2>
+            {{ 'Reference'|trans({}, 'Admin.Catalog.Feature') }}
+            <span class="help-box"
+                  data-toggle="popover"
+                  data-content="{{ 'Your reference code for this product. Allowed special characters: .-_#\.'|trans({}, 'Admin.Catalog.Help') }}" >
+            </span>
+          </h2>
+          <div class="row">
+            <div class="col-xl-12 col-lg-12">
+              {{ form_widget(productForm.basic_settings.reference) }}
+            </div>
+          </div>
+        </div>
+
+        {% if 'PS_STOCK_MANAGEMENT'|configuration %}
+          <div class="form-group mb-4">
+            <h2>
+              {{ 'Quantity'|trans({}, 'Admin.Catalog.Feature') }}
+              <span class="help-box"
+                    data-toggle="popover"
+                    data-content="{{ 'How many products should be available for sale?'|trans({}, 'Admin.Catalog.Help') }}" >
+              </span>
+            </h2>
+            <div class="row">
+              <div class="col-xl-6 col-lg-12">
+                {{ form_widget(productForm.basic_settings.quantity) }}
+              </div>
+            </div>
+            <span class="small font-secondary">
+              {{ 'Advanced settings in [1][2]Quantities[/1]'|trans({}, 'Admin.Catalog.Help')|replace({'[1]': '<a href="#tab-step3" onclick="$(\'a[href=\\\'#step3\\\']\').tab(\'show\');" class="btn sensitive px-0">', '[/1]': '</a>', '[2]': '<i class="material-icons">open_in_new</i>'})|raw }}
+            </span>
+          </div>
+        {% endif %}
+
+        <div class="form-group mb-4">
+          <h2>
+            {{ 'Price'|trans({}, 'Admin.Global') }}
+            <span class="help-box"
+                  data-toggle="popover"
+                  data-content="{{ 'This is the retail price at which you intend to sell this product to your customers. The tax included price will change according to the tax rule you select.'|trans({}, 'Admin.Catalog.Help') }}" >
+            </span>
+          </h2>
+          <div class="row">
+            <div class="col-md-6">
+              <label class="form-control-label">{{ "Tax excluded"|trans({}, 'Admin.Catalog.Feature') }}</label>
```

Comment:
Are you sure it is the right commas?

Reply:
What do you mean?

Reply:
What do you mean?

Reply:
Here you are using the **"** type of commas and I often read strings between the **'** type of commas. Anyway, I was just raising this remark in case it was an error.

Reply:
Ohhh, no it's not a problem. :)